tests/integration/conftest.py

```diff
@@ -0,0 +1,22 @@
+"""Fixtures for all integration tests"""
+import os
+import subprocess
+import pytest
+
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_and_teardown_server():
+    """ Sets up a server for tests and shut it down when it is finished"""
+    # Run server
+    server_file = os.path.abspath(os.path.join(os.path.dirname(__file__),
```

Comment:
you need to fix server setup. there wouldn't be a server file here. you should be installing the server as a pkg.